oneandone/src/main/java/org/apache/jclouds/oneandone/rest/features/VpnApi.java

```diff
@@ -62,9 +65,10 @@
    @Named("vpn:configurations:get")
    @GET
    @Path("/{vpnId}/configuration_file")
-   @SelectJson("config_zip_file")
+   @ResponseParser(VPNConfigParser.class)
```

Comment:
Is this parser class needed? You could leave the `@SelectJson` annotation and just create the transformation function as a top level class.